views/requests/index.erb

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Document</title>
+</head>
+<body>
+  <div>
+    <h2>Requests I've made</h2>
+    <ul>
+      <% @requests.each do |request| %>
+        <li class='request'>
+         <h3><%= Space.find(request.space_id).name %></h3>
```

Comment:
Probably we should use an instance variable and this should be moved to the controller.